src/richie/apps/courses/settings/__init__.py

```diff
@@ -304,6 +304,11 @@ def richie_placeholder_conf(name):
         "plugins": ["CKEditorPlugin"],
         "limits": {"CKEditorPlugin": 1},
     },
+    "courses/cms/category_detail.html course_faq": {
```

Comment:
Replace with:

```yaml
    "courses/cms/category_detail.html additional_information": {
        "name": _("Additional Information"),
        "plugins": ["SectionPlugin"],
        "parent_classes": {
            "CKEditorPlugin": ["SectionPlugin"],
            "SimplePicturePlugin": ["SectionPlugin"],
            "GlimpsePlugin": ["SectionPlugin"],
            "NestedItemPlugin": ["SectionPlugin"],
        },
        "child_classes": {
            "SectionPlugin": ["CKEditorPlugin", "SimplePicturePlugin", "GlimpsePlugin", "NestedItemPlugin"],
            "NestedItemPlugin": ["NestedItemPlugin"],
        },
    },
```

To it has a similar behavior of the `course_information` placeholder of the `course_detail.html` page.
But to also include the `NestedItemPlugin` that allows to show a nice FAQs accordion.
So this placeholder can have any Section (title h2) naming. It could be FAQs, Additional Information, My complementary information, etc.